README.md

```diff
@@ -25,7 +75,7 @@ GCC >= 8 is required to build Redis on Unikraft.
 On Ubuntu/Debian or other `apt`-based distributions, run the following command to install the requirements:
 
 ```console
-$ sudo apt install -y --no-install-recommends \
+sudo apt install -y --no-install-recommends \
```

Comment:
Add `redis-tools` as install package. It provides `redis-cli` command line tool.